dockerfiles/Dockerfile-build-crypt

```diff
@@ -87,7 +87,7 @@ RUN export EMCC_CFLAGS="-I/emsdk/upstream/include -L/emsdk/upstream/lib" \
     && emmake make
 
 # Build UI with encryption
-FROM node:16.15.1-alpine3.15 as FRONTEND
+FROM node:19.0.0-alpine3.15 as FRONTEND
```

Comment:
are you sure you want to go node:19:00 ?

Reply:
Good catch, managed to miss that somehow 😅 We most definitely don't, but I'll see if 18 (current LTS) builds